pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_activation/guide.fr-fr.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Guides premiers pas avec IAM dans mon cloud privée VMware managé par OVHcloud"
+excerpt: "Comment activer avec IAM dans une infrastructure VMware managé par OVHcloud"
+updated: 2024-05-24
+---
+
+# Table des matières
```

Comment:
This index is not necessary. We only add it on very long guides. Please remove it.